### PR TITLE
ECIP for Communicating with 3rd parties

### PR DESCRIPTION
**Pull request by bobthegrownup**
This proposal is a framework for 3rd party communications in decentralized systems

### CODE REVIEW

File: ECIP-dean-comms [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Communicating with 3rd parties
+
+    ECIP: ?
```
Comment (soc1c) [inline ECIP-dean-comms]:
```suggestion
    ECIP: 1064
```

**Comment by sorpaas** [comment]
@bobthegrownup Fixed some formatting issues for you. Haven't yet finished reading it!

File: _specs/ecip-1064.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+---
+lang: en
+ecip: 1064
+title: Communicating with 3rd parties
+status: Draft
+type: Informational
```
Comment (sorpaas) [inline _specs/ecip-1064.md]: This probably should be "Meta". We don't have a "Informational" category at this moment.

```suggestion
type: Meta
```

File: _specs/ecip-1064.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+1) Representative Selection
+2) Message Construction
+3) Channels
+
+### Representative Selection
+
+Each Client (geth/multigeth/parity/mantis) should have 1 representative selected to help in the outreach process.  In addition, there should be 2 representatives voted by the Client representatives that are unaffiliated with any of the Organizations.
+
+This selection happens at the “Final Call” of the event.
```
Comment (sorpaas) [inline _specs/ecip-1064.md]: Maybe specify that this only applies to "Final Call" of Hard Fork Meta ECIPs. ECIP has a status called "Final Call", and I don't think all of them will need to go through this process.

Comment (bobthegrownup) [reply _specs/ecip-1064.md]: Totally agree

Comment (YazzyYaz) [reply _specs/ecip-1064.md]: @bobthegrownup preferably before we merge it. Let me know if you need help with that.

File: _specs/ecip-1064.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+*For example, at the Final Call of Atlantis, the following representatives are selected:*
+1. ETCLabs – Christian
```
[the remark this answers is not in the document]
Comment (bobthegrownup) [reply _specs/ecip-1064.md]: Yes, the client is a much better choice

File: _specs/ecip-1064.md (hunk 3 above)
Comment (bobthegrownup) [reply _specs/ecip-1064.md]: @sorpaas @YazzyYaz I added the Hard Fork detail for ECIPs and updated the proposal with the concept of "Black Swan" events. The focus is still with hard forks, but it now encompasses the emergency events.

**Comment by YazzyYaz** [comment]
Approving since I'm satisfied with the "Black Swan" choice of words.

**Pull request merged**